### PR TITLE
Bm benchmarking challenge pages

### PR DESCRIPTION
**Pull request by Bianca-Morris**
**Changelog:**
* Add new section to Data nav for challenge results
* Add COLO829 SNV/Indel V1 Challenge Page + tables

This is viewable on devtest as of July 26th, 2024: https://devtest.smaht.org/data/analysis/colo829-snv-indel-detection

### CODE REVIEW

**Comment by Bianca-Morris** [comment]
[Changes to Navigation - adding Resources Tab](https://trello.com/c/eefwoLPM/291-changes-to-navigation-adding-resources-tab)

**Comment by Bianca-Morris** [comment]
[COLO829 SNV/Indel Challenge Page](https://trello.com/c/gxMkszyg/295-colo829-snv-indel-challenge-page)

**Review by crfmc — CHANGES_REQUESTED** [review]
The new page's navigation link has a different width and hover state. It looks like it may be the width of the parent.

Screenshot 2024-07-31 at 2 43 51 PM

File: src/encoded/static/scss/encoded/modules/_static-pages.scss [hunk 1]
```diff
@@ -1727,6 +1727,16 @@ body.is-item-view[data-pathname^="/users/"] #page-title-container,
 	border-right: none;
 }
 
+.search-result-row.detail-closed >.columns.result-table-row>.search-result-column-block {
+	&[data-first-visible-column=true] {
+		padding-left: 16px !important;
```
[the remark this answers is not in the document]
Comment (crfmc) [reply src/encoded/static/scss/encoded/modules/_static-pages.scss:1732]: If overwriting, updating these lines in [_item-pages.scss ](https://github.com/smaht-dac/smaht-portal/blob/9f57189affb2dd87e578a1fbdb2bb9a572fe0e45/src/encoded/static/scss/encoded/modules/_item-pages.scss#L1626-L1628) should do the trick.

```
&[data-first-visible-column=true] {
     padding-left: 0!important;
}
```

[the remark this answers is not in the document]
Comment (Bianca-Morris) [reply src/encoded/static/scss/encoded/modules/_static-pages.scss]: This should be fixed

**Review by crfmc — APPROVED** [review]

**Comment by Bianca-Morris** [comment]
This should not be merged as-is currently. The styling for the modal is not appearing for the challenge page at the moment. We'll need to work out a fix for this when I return.

[Image from Gyazo](https://gyazo.com/83f4ef6494b5230a867892bbdda054c9)

**Pull request merged**